robotbase/src/main/java/frc/robot/controls/DriverControls.java

```diff
@@ -32,7 +33,7 @@ public DriverControls(XboxController controller, double deadband) {
 
     m_rightTriggerPrime = new AxisThresholdTrigger(m_controller, Axis.kRightTrigger, .1);
     m_rightTriggerShoot = new AxisThresholdTrigger(m_controller, Axis.kRightTrigger, .6);
-    m_leftTrigger = new AxisThresholdTrigger(m_controller, Axis.kLeftTrigger, .1);
+    m_leftTrigger = new AxisThresholdTrigger(m_controller, Axis.kLeftTrigger, 0);
```

Comment:
We do not want this to be zero. Axis's on controllers are never truly at zero, they usually have a small amount of jitter meaning that a threshold of zero could cause this command to run. For example, the axis could jitter to a value of 0.000001 which will cause the command to run.